app/(tabs)/explore.tsx

```diff
@@ -1,97 +1,63 @@
-import { StyleSheet, Image, Platform } from 'react-native';
-
-import { Collapsible } from '@/components/Collapsible';
-import { ExternalLink } from '@/components/ExternalLink';
-import ParallaxScrollView from '@/components/ParallaxScrollView';
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Image, Platform, View, Text, FlatList } from 'react-native';
+import { getExercises} from '../../api/wgerAPI';
 import { ThemedText } from '@/components/ThemedText';
 import { ThemedView } from '@/components/ThemedView';
-import { IconSymbol } from '@/components/ui/IconSymbol';
+import { Exercise } from '@/.expo/types/exercise';
```

Comment:
The interface you had made before should just be in a new folder types/exercise.ts becuase I'm not sure how reliable the .expo types are